test/integration-tests/testset_sdk/javascript/testSetSDK.js

```diff
@@ -31,12 +33,76 @@ class TestSetSDK extends TestSetTalent {
         this.registerTest('echoMixedList', [1, 'Hello World', 3.21], this.test_echoMixedList.bind(this), 5000);
         this.registerTest('echoDeepList', [1, [2, [3, [4, [5]]]]], this.test_echoDeepList.bind(this), 5000);
 
+        this.registerTest('receiveEvent1', true, this.test_receiveEvent1.bind(this), 5000);
+
         // Add external talentIds which this test set is depending on
         this.talentDependencies.addTalent('function-provider-js');
+        this.talentDependencies.addTalent('event-tester-1-js');
+        this.talentDependencies.addTalent('event-tester-2-js');
+
+        this.addOutput("receive_event_1", {
+            "description": "receiveEvent1",
+                "idx": 1,
+                "encoding": {
+                    "type": "string",
+                    "encoder": null
+                }
+            });
     }
 
     callees() {
-        return [ 'function-provider-js.echo' ];
+        return [
+            'function-provider-js.echo',
+            'event-tester-1-js.get_received_events',
+            'event-tester-2-js.get_received_events',
+        ];
+    }
+
+    async test_receiveEvent1(ev, evtctx) {
```

Comment:
I propose to provide a more detailed test name (e.g. test_receiveEvent1ByMultipleTalents)